### ISSUE TITLE
need exclusion skin, for traders

### ISSUE THREAD

**Issue by F507DMT** [dayzai/DZAI#12]
need exclusion skin, for traders.
ai all time shoot in traders

**Comment by Rythron** [comment]
yeah i am trying to figure that one out for about 6 months or longer now, tried a lot of things. Has anybody got a solution for this?

**Comment by F507DMT** [comment]
http://epochmod.com/forum/index.php?/topic/33622-releasefix-ai-try-kill-traders/

**Comment by F507DMT** [comment]
i found fix for this
